docs/labwc-config.5.scd

```diff
@@ -344,7 +344,7 @@ Therefore, where multiple objects of the same kind are required (for example
 	- Desktop: The desktop background, where no windows are present.
 	- Root: A synonym for Desktop (for compatibility).
 
-	Supported mouse *buttons* are:
+	Supported mouse *buttons* can be combined with modifiers and are:
```

Comment:
Should we say `keyboard modifiers` here to make it clear what 'modifiers' means in this context?

Reply:
Considering mentioning modifier-keys in a separate paragraph for simplicity and clarity.

For example:

```
Mouse buttons can be combined with modifier-keys (shift, super, control and alt), for example:

    <mousebind button="A-Right" action="Press">

```